pkg/streamingpromql/engine_test.go

```diff
@@ -1786,6 +1786,10 @@ func TestCompareVariousMixedMetrics(t *testing.T) {
 	for _, labels := range labelCombinations {
 		labelRegex := strings.Join(labels, "|")
 		// Aggregations
+		// TODO(jhesketh): Add stddev back ing.
+		// stddev is excluded until https://github.com/prometheus/prometheus/pull/14941 is merged
+		// fixing an inconsistency in the promql engine where if a native histogram is the first sample
+		// loaded, it is incorrectly treaded as a 0 float point.
```

Comment:
```suggestion
		// TODO(jhesketh): Add stddev back in.
		// stddev is excluded until https://github.com/prometheus/prometheus/pull/14941 is merged
		// fixing an inconsistency in Prometheus' engine where if a native histogram is the first sample
		// loaded, it is incorrectly treated as a 0 float point.
```

Reply:
I don't own a cat. I've had coffee. I don't know why I type bad and such.